File: praticas_1/pratica1.py
PI = 3.14  

def calcular_circulo(raio):
    comprimento = 2 * PI * raio
    area = PI * raio ** 2
    return comprimento, area

def calcular_retangulo(lado1, lado2):
    comprimento = 2 * (lado1 + lado2)
    area = lado1 * lado2
    return comprimento, area
    
print("Escolha uma opcao:")
print("1 - Calcular a area e o comprimento de um circulo")
print("2 - Calcular a area e o perimetro de um retangulo")

opcao = input("Digite o numero da opcao desejada: ")

if opcao == "1":
    print("\nCalculando a area e o comprimento de um circulo")
    raio = float(input("Informe o valor do raio: "))
    comprimento, area = calcular_circulo(raio)
    print(f"Comprimento: {comprimento:.2f}")
    print(f"Area: {area:.2f}")

elif opcao == "2":
    print("\nCalculando a area e o perimetro de um retangulo")
    lado1, lado2 = map(float, input("Digite os lados do retangulo (Lado1, Lado2): ").split(','))
    perimetro, area = calcular_retangulo(lado1, lado2)
    print(f"O perimetro do retangulo e: {perimetro:.2f}")
    print(f"A area do retangulo e: {area:.2f}")

else:
    print("Opcao invalida. Escolha 1 ou 2.")

File: praticas_1/pratica1.cpp
/*

Program: [C++]

Author: [Fabiana Queren]

Date: [15/03/2025]

Description: [Calculo da area e contorno de um circulo e um retangulo]

License: [CC BY-NC-SA ]

*/

#include <iostream>
#define PI 3.14  

using namespace std;

void calcular_circulo(float raio, float &comprimento, float &area) {
    comprimento = 2 * PI * raio;
    area = PI * raio * raio;
}

void calcular_retangulo(float lado1, float lado2, float &perimetro, float &area) {
    perimetro = 2 * (lado1 + lado2);
    area = lado1 * lado2;
}

int main() {
    int opcao;
    float raio, comprimento, area, lado1, lado2, perimetro;

    cout << "Escolha uma opcao:" << endl;
    cout << "1 - Calcular a area e o comprimento de um circulo" << endl;
    cout << "2 - Calcular a area e o perimetro de um retangulo" << endl;
    cout << "Digite o numero da opcao desejada: ";
    cin >> opcao;

    if (opcao == 1) {
        cout << "\nCalculando a area e o comprimento de um circulo" << endl;
        cout << "Informe o valor do raio: ";
        cin >> raio;

        calcular_circulo(raio, comprimento, area);

        cout << "Comprimento: " << fixed << comprimento << endl;
        cout << "Area: " << fixed << area << endl;
    } 
    else if (opcao == 2) {
        cout << "\nCalculando a area e o perimetro de um retangulo" << endl;
        cout << "Digite os lados do retangulo separados por vírgula e espaço (Lado1, Lado2): ";
        
        char separador;
        cin >> lado1 >> separador >> lado2;

        calcular_retangulo(lado1, lado2, perimetro, area);

        cout << "O perimetro do retangulo e: " << fixed << perimetro << endl;
        cout << "A area do retangulo e: " << fixed << area << endl;
    } 
    else {
        cout << "Opcao invalida. Escolha 1 ou 2." << endl;
    }

    return 0;
}